app/components/admin/users-table.tsx

```diff
@@ -93,13 +94,18 @@ export default function UsersTable({ users, searchText }: UsersTableProps) {
                             />
                           </Form>
                         </Tooltip>
-                        <Tooltip label="Deactivate user">
-                          <IconButton
-                            aria-label="Deactivate user"
-                            icon={<DeleteIcon color="black" boxSize={5} />}
-                            variant="ghost"
-                          />
-                        </Tooltip>
+                        <Form method="post" reloadDocument>
+                          <Tooltip label="Deactivate user">
```

Comment:
I'd use `Delete` vs. `Deactivate` everywhere, since we're really dropping the user from the db.